### PR TITLE
doc: update links to point to ceph/qa instead of ceph-qa-suite

### PR DESCRIPTION
**Pull request by jan--f**
Also fix two broken links to install task.

Signed-off-by: Jan Fajerski <jfajerski@suse.com>

### CODE REVIEW

**Comment by jan--f** [comment]
@smithfarm you wanted to have a look?

File: doc/dev/development-workflow.rst [hunk 1]
```diff
@@ -139,15 +139,15 @@ results are posted on `pulpito <http://pulpito.ceph.com/>`_ and the
...
-* `rgw <https://github.com/ceph/ceph-qa-suite/tree/master/suites/rgw>`_ suite
-* `CephFS <https://github.com/ceph/ceph-qa-suite/tree/master/suites/fs>`_ suite
-* `rados <https://github.com/ceph/ceph-qa-suite/tree/master/suites/rados>`_ suite
-* `rbd <https://github.com/ceph/ceph-qa-suite/tree/master/suites/rbd>`_ suite
+* `rgw <https://github.com/ceph/ceph/tree/master/qa/suites/rgw>`_ suite
+* `CephFS <https://github.com/ceph/ceph/tree/master/qa/suites/fs>`_ suite
+* `rados <https://github.com/ceph/ceph/tree/master/qa/suites/rados>`_ suite
+* `rbd <https://github.com/ceph/ceph/tree/master/qa/suites/rbd>`_ suite
 
 and a many others such as
```
Comment (smithfarm) [inline doc/dev/development-workflow.rst]: and many others (drop the indefinite article)

**Comment by smithfarm** [comment]
@jan--f: LGTM. I pushed a commit modifying the very last section of `doc/dev/index.rst` - please take a look.

**Comment by liewegas** [comment]
retest this please

**Comment by jan--f** [comment]
@smithfarm looks good. Test failures seem unrelated.

**Comment by smithfarm** [comment]
@jan--f Looks like make check is failing on all master PRs atm.

**Comment by dmick** [comment]
include fc28dcaed5f7f358718f22ac7e3b1ef79de3330f

**Comment by smithfarm** [comment]
@dmick Thanks, rebased to grab that.

**Comment by dmick** [comment]
someone want to merge this?

File: doc/dev/index.rst [hunk 2]
```diff
@@ -1337,21 +1337,21 @@ manually.
...
 adapt a test to stop and wait after the Ceph installation phase, and the
-engineer can then ssh into the running cluster.
+engineer can then ssh into the running cluster. Simply add the following
+snippet in the desired place within the test YAML and schedule a run with the
+test::
 
-This common use case is currently provided for by the following command::
+    - exec:
+        client.0:
+          - sleep 1000000000 # forever
```
Comment (tchaikov) [inline doc/dev/index.rst]: would be 
```
tasks:
 - interactive
```
a better choice?

Comment (smithfarm) [inline doc/dev/index.rst]: @tchaikov Good idea! I read the documentation at https://github.com/ceph/teuthology/blob/master/teuthology/task/interactive.py#L13-L29 from which I suppose that this causes an interactive Python shell to be run, but I'm not sure how to connect to that shell?

Comment (tchaikov) [inline doc/dev/index.rst]: i am able to have the interactive python shell using following command in sepia.
```
teuthology ~/test.yaml --lock --machine-type mira
```

where `test.yaml` is

```yaml
roles:
    - [client.0]
repo: git://git.ceph.com/ceph.git
tasks:
    - interactive:
```

**Comment by smithfarm** [comment]
@tchaikov I added a commit mentioning the `interactive` task. Please take a look.

**Pull request merged**